presto-mixin/dashboards/presto-worker-overview.libsonnet

```diff
@@ -0,0 +1,957 @@
+        lineWidth: 2,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'normal',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'none',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'right',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local memoryPoolPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum by (instance, presto_cluster) (com_facebook_presto_memory_MemoryPool_FreeBytes{job=~"$job", presto_cluster=~"$presto_cluster", instance=~"$instance"})',
+      datasource=promDatasource,
+      legendFormat='{{presto_cluster}} - {{instance}} - free',
+      format='time_series',
+    ),
+    prometheus.target(
+      'sum by (instance, presto_cluster) (com_facebook_presto_memory_MemoryPool_ReservedBytes{job=~"$job", presto_cluster=~"$presto_cluster", instance=~"$instance"})',
+      datasource=promDatasource,
+      legendFormat='{{presto_cluster}} - {{instance}} - reserved',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Memory pool',
+  description: 'The amount of Presto memory available.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisBorderShow: false,
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 15,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        insertNulls: false,
+        lineInterpolation: 'smooth',
+        lineWidth: 2,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'normal',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'decbytes',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'right',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local dataThroughputPanel = {
```

Comment:
Can you double check the metric and unit for this? it's using that `TotalCount` metric and it's not obvious if it's correct

Reply:
👍 Changed the metric to be using OneMinute_Rate, units should be good. Also changed graph type to line with 90 degrees since.

Reply:
Same comment here re: naming of the panel, I think "data processing throughput" or similarly descriptive while dropping "input/output" will work.

Similarly, can you check what unit the metric is? Same reasoning as the other panel re: bytes & millibytes